src/main/java/edu/project2/tripplanner/controller/TripController.java

```diff
@@ -0,0 +1,70 @@
+package edu.project2.tripplanner.controller;
+
+import edu.project2.tripplanner.model.Trip;
+import edu.project2.tripplanner.service.ITripService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RestController
+@AllArgsConstructor
+public class TripController {
+
+    ITripService iTripService;
+
+    @ResponseBody
+    @GetMapping("/trip/{userId}")
+    public List<Trip> findTripsByUserId(@PathVariable(value = "userId") Long userId) {
+
+        return iTripService.findAllByUserId(userId);
+    }
+
+
+    @ResponseBody
+    @PostMapping("/user/{userId}/trip")
```

Comment:
in controller Trip shouldn't be 'user' at all. If you use 'userId' parameter - it's enough